src/components/Navbar/Navbar.vue

```diff
@@ -0,0 +1,1259 @@
+	align-items: center;
+	width: 95%;
+	margin-top: 35px;
+	z-index: 60;
+	transition: box-shadow 180ms, background 180ms;
+}
+
+header * {
+	z-index: 1;
+}
+
+header::before {
+	content: "";
+	position: absolute;
+	top: -35px;
+	left: -10vw;
+	width: 120vw;
+	height: calc(100% + 35px + 35px);
+	background: rgba(27, 31, 59, 0.98);
+	transition: background 180ms;
+}
+
+header.transparent,
+header.transparent::before {
+	background: rgba(27, 31, 59, 0);
+}
+
+header.dropdown-active {
+	background: rgba(27, 31, 59, 0.98);
+	box-shadow: 0 0 0 600vw rgba(27, 31, 59, 0.8);
+}
+
+header .dropdown-arrow {
+	opacity: 0;
+	position: absolute;
+	width: 0;
+	border-style: solid;
+	border-color: var(--bg-shade-3) transparent;
+	border-width: 0 14px 14px;
+	bottom: -26px;
+	margin-left: -10px;
+	margin-bottom: -10px;
+	transition: left 180ms, margin-bottom 180ms, opacity 180ms;
+	pointer-events: none;
+}
+
+header.dropdown-active .dropdown-arrow {
+	opacity: 1;
+	display: block;
+	margin-bottom: 0;
+}
+
+header a {
+	text-decoration: none;
+}
+
+header .logo-link,
+header .logo-link svg {
+	display: block;
+}
+
+header div.left-section {
+	display: flex;
+	flex-flow: row nowrap;
+}
+
+header nav {
+	position: relative;
+	display: flex;
+	flex-flow: row nowrap;
+	align-items: center;
+	margin-left: 40px;
+}
+
+header button.dropdown-button#mobile-button {
+	display: none;
+	background: none;
+	padding: 0;
+	margin-right: 12px;
+	transition: transform 200ms;
+}
+
+header button.dropdown-button#mobile-button.active {
+	transform: rotate(90deg);
+}
+
+header div.dropdown-button-wrapper {
+	position: relative;
+	display: flex;
+	flex-flow: row nowrap;
+}
+
+/* these are safezones where the dropdown will not close */
+header div.dropdown-button-wrapper::before {
+	content: "";
+	position: absolute;
+	top: -16px;
+	bottom: 100%;
+	left: 0;
+	right: -12px;
+}
+
+header div.dropdown-button-wrapper::after {
+	content: "";
+	position: absolute;
+	top: 100%;
+	bottom: -32px;
+	left: 0;
+	right: -12px;
+}
+
+header nav button {
+	background: none;
+	color: var(--text-shade-1);
+	margin: 0 17px;
+	padding: 0;
+}
+
+header nav button:hover,
+header nav button.active {
+	background: none;
+	color: var(--text-shade-3);
+}
+
+header nav a.donate button {
+	display: grid;
+	grid-auto-flow: column;
+	align-items: center;
+	gap: 4px;
+	font-weight: bold;
+	background: #332b61;
+	color: var(--accent-shade-3);
+	padding: 2px 12px;
+	border-radius: 24px;
+}
+
+header nav a.donate button svg {
+	height: 1rem;
+	width: 1rem;
+}
+
+header nav a.donate button:hover {
+	background: var(--accent-shade-0);
+	color: #fff;
+}
+
+header nav a.donate button.dropdown-button::after {
+	content: "";
+	display: inline-block;
+	align-items: center;
+	width: 16px;
+	height: 8px;
+	bottom: 0;
+	margin-left: 0.3rem;
+	background: no-repeat center url("/assets/images/down-arrow.svg");
+	filter: brightness(0) invert(78%) sepia(2%) saturate(5488%) hue-rotate(197deg) brightness(88%) contrast(93%);
```

Comment:
The image is missing, let's also copy it into the project (should be fine to drop it into the [public directory](https://nuxt.com/docs/guide/directory-structure/public))